scripts/eia/Rwanda-RAB-AddOn.R

```diff
@@ -61,7 +60,7 @@ carob_script <- function(path) {
       #previous_crop=r$crop_cultivated_all,
       fertilizer_amount= as.numeric(r$fertiliser_amount),
       fertilizer_type= r$fertiliser_type,
-      irrigation_dates= r$Irrigation_months, 
```

Comment:
Why remove? It can be captured by standardizing the dates to the format [here](https://github.com/reagro/terminag/blob/3cb74d2a3954f467554365441d32985fbb83b41d/variables/variables_crop.csv#L78)